docs/concepts/usage-ideas-examples.md

```diff
@@ -2,14 +2,317 @@
+
+## IPFS as infrastructure
+
+Using IPFS allows you to abstract away a lot of the complexity of coordination between machines. No matter your architecture IPFS handles load balancing, de-duplication, caching, and high availability out-of-the-box. It's highly modular design also means you can easily customize it to your need.
+
+### Decentralized clusters
+
+[IPFS Cluster](https://cluster.ipfs.io/) is the official tool to manage a cluster of nodes to replicate data. Like any distributed cluster, you will have the benefits of redundancy, load balancing, and write permission management. You can choose to connect your cluster to the rest of the IPFS network or run it privately. You can also invite outsiders to help replicate your data without them having write-access, thanks to the [collaborative mode](https://cluster.ipfs.io/documentation/collaborative/).
+
+### Content delivery networks
+
+A [Content Delivery Network](https://en.wikipedia.org/wiki/Content_delivery_network) (CDN) is a network of nodes storing content in places physically close to the users. Having servers physically closer to the users ensures lower latency, enabling load-balancing, and allows you to scale the availability of content with demand. The IPFS network is a CDN by design because each node will cache what they consume and serve that data to its peers.
+
+### Distributed package managers
+
+The [Node package manager (NPM)](https://www.npmjs.com/) is mirrored on IPFS. By using the dedicated client [npm-on-ipfs](https://github.com/ipfs-shipyard/npm-on-ipfs), packages are fetched from IPFS and distributed to other clients needing them. For example, a team working in the same building will fetch packages from each other, meaning less network traffic costs for the company.
+
+### Hosting software containers
+
+Netflix is on its way to use IPFS to [synchronize their Docker containers worldwide](https://blog.ipfs.io/2020-02-14-improved-bitswap-for-container-distribution/). Since each node fetches them from the fastest peers they know, synchronization is substantially faster than regular sync methods.
+
+### Efficient network factories
+
+[Actyx](https://www.actyx.com/) is helping the manufacturers to fast forward to what they're calling _Industry 4.0_: the industry of connected machines. Actyx built a custom operating system on top of IPFS to ensures all machines and monitors receive the orders, information, and parameters they need to function properly.
```

Comment:
Industry 4.0 concept is not Actyx's invention, it's a known concept in the manufacturing game. Maybe linking to Wikipedia? https://en.wikipedia.org/wiki/Industry_4.0

Reply:
The term _Industry 4.0_ isn't particularly common place. Maybe you could expand upon what Industry 4.0 is before this paragraph.

Reply:
Done